.travis.yml

```diff
@@ -1,6 +1,12 @@
+dist: trusty
```

Comment:
Travis uses Ubuntu Precise as a default image, but with this option Ubuntu Trusty will be used. So official support of Precise is ending, and as I know, Travis is looking to change default image to Trusty in Q3 2017 

You can get some info [here](https://docs.travis-ci.com/user/trusty-ci-environment#Using-Trusty)